sdk/messaging/azservicebus/internal/mgmt.go

```diff
@@ -49,7 +49,7 @@ type (
 
 type MgmtClient interface {
 	Close(ctx context.Context) error
-	SendDisposition(ctx context.Context, lockToken *amqp.UUID, state Disposition) error
+	SendDisposition(ctx context.Context, lockToken *amqp.UUID, state Disposition, propertiesToModify *map[string]interface{}) error
```

Comment:
I know this is internal but why is it a pointer?

Reply:
Well, now I just feel silly. I'll remove (clearly map[] can be nil!). Fixed.